instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
@antoviaque The active AppServer will be shown as "terminated" in the UI, so it shouldn't be surprising it can't be reached.  If you prefer, we could also show the "please wait" screen again in this situation, but in general if the active AppServer is terminated this means that the instance has been shut down.